src/js/view/reducers/logInfoReducer.js

```diff
@@ -9,20 +9,17 @@ export const logInfoReducer = (state = initialState, action) => {
       };
     case 'LOGINFO_INCREASE_SIZE': {
```

Comment:
Missing tests for this very testable file :)

Reply:
Såklart det ska testas! :swimming_man: